script/Ciel/Day 8/08_01_CIEL08_6.txt

```diff
@@ -657,25 +701,27 @@ The sixth ceremony had begun.
 [sha:3ca376f6e6428a1c04a14000e84bd8e9849985ad]{
 -- Page 541, Offset 19658.
 -- 『私』はわたしという肉体を得て、ようやくあの白い女を手に入れられると判断し、そして―――
-'I' obtained the body I once possessed, then judged that 'I' would finally be able to make the woman in white mine―――
+Having obtained the body that had once been mine,
+ 'I' thought that they'd finally be able to make the woman in white mine, and then―――
 }
 [sha:d62390bde028dbf1f7d9b7efc6f972c17dd5f913]{
 -- Page 542, Offset 19659.
 -- ……そして、あってはならない事変は収束した。
-...Then, the incident that should not have been came to an end.
+And then... the incident that should never have been came to an end.
 }
 [sha:9bcff5ad614a407661ad60e49ca4532307a1dd1a]{
 -- Page 542, Offset 19660.
 -- 戦いの末、あの白い吸血姫は『私』を仕留めたのだ。
-At the end of the battle, that white vampire princess killed 'me'.
+At the end of the battle, the vampire princess in white killed 'me'.
 }
 [sha:481c251ba9a31c6010ae1afd23db9aba064d976e]{
 -- Page 542, Offset 19661.
 -- すでに転生の準備を進めていた『私』はまたも転生し、残されたものはわたしの遺体だけ。
-'I' reincarnated once again, having already made the preparations for it, leaving behind my corpse.
+'I' had already made preparations for their next reincarnation and was born again,
+ leaving behind nothing but my corpse.
```

Comment:
pronoun weirdness again, though I suppose we'll have to do a general overview of how to deal with this.

(it returns later again)

Reply:
I think the pronouns are correct on this one? At least for the fucky kind of correct that this scene entails? 'Roa' had already made preparations ... and was born again, leaving behind nothing but Ciel's corpse.